config/filesystems.php

```diff
@@ -41,12 +41,38 @@
 	*/
 
 	'disks' => [
+		'local' => [
+			'driver' => 'local',
+			'root' => storage_path('app'),
+			'throw' => true,
+		],
+
+		'public' => [
+			'driver' => 'local',
+			'root' => storage_path('app/public'),
+			'url' => env('APP_URL') . '/storage',
+			'visibility' => 'public',
+			'throw' => true,
+		],
+
+		's3' => [
+			'driver' => 's3',
+			'key' => env('AWS_ACCESS_KEY_ID'),
+			'secret' => env('AWS_SECRET_ACCESS_KEY'),
+			'region' => env('AWS_DEFAULT_REGION'),
+			'bucket' => env('AWS_BUCKET'),
+			'url' => env('AWS_URL'),
+			'endpoint' => env('AWS_ENDPOINT'),
+			'use_path_style_endpoint' => env('AWS_USE_PATH_STYLE_ENDPOINT', false),
+			'throw' => true,
+		],
+
 		// Lychee uses the disk "images" to store the media files
 		'images' => [
 			'driver' => 'local',
 			'root' => env('LYCHEE_UPLOADS', public_path('uploads/')),
 			'url' => env('LYCHEE_UPLOADS_URL', 'uploads/'),
-			'visibility' => env('LYCHEE_IMAGE_VISIBILITY', 'public'),
+			'visibility' => 'public',
```

Comment:
Why that? We just made it configurable some PR ago.

Reply:
Fixed [bb29f73](https://github.com/LycheeOrg/Lychee/pull/1469/commits/bb29f730217eb210ec0f538355080e183a74fe0c)